### DISCUSSION TITLE
Enhancement suggestion for Access Control

### DISCUSSION THREAD

**Discussion by emmaparker96** [proginosko/LeechBlockNG#380] [Ideas]
Currently, the selected Access Control to the options is always requested, regardless of whether a Blockset is currently active or not. This can be frustrating when you want to adjust the options during your time off.

Therefore I suggest the following to allow this optional:

![Screenshot 2020-10-21 121550](https://user-images.githubusercontent.com/66069567/96707252-e8b38280-1397-11eb-8930-80081ce33b08.png)

**Comment by proginosko** [comment]
Good suggestion. Perhaps even better would be to include a buffer time (e.g., request access control only if block will be active within __ minutes).
